### PR TITLE
Add, configure & enforce sbt-whitesource

### PR DESCRIPTION
**Pull request by dwijnand**

### CODE REVIEW

**Comment by typesafe-tools** [comment]
To validate this pull request against other sbt modules, please visit <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/parambuild/?zinc=pull/482/head&deploy_to_bintray=false&run_sbt_community=false">this link.</A>

**Comment by jvican** [comment]
Can you explain a little bit what this plugin is?

File: build.sbt [hunk 1]
```diff
@@ -621,3 +621,12 @@ def customCommands: Seq[Setting[_]] = Seq(
       state
   }
 )
+
+inThisBuild(Seq(
+  whitesourceProduct                   := "Lightbend Reactive Platform",
```
Comment (jvican) [inline build.sbt:626]: Does this mean you're offering subscriptions for this product? This is novel, it's not even the case for scalac (at least there's no references to the LRP in the sources). Please, can you clarify?

**Comment by dwijnand** [comment]
The plugin sends dependency information to the central server, which we use to valid things like licenses.

File: build.sbt (hunk 1 above)
Comment (dwijnand) [reply build.sbt:626]: Yeah scala, sbt, zinc are products of RP, and Lightbend provides commercial support for them.

**Review by jvican — APPROVED** [review]

**Comment by dwijnand** [comment]
Cheers.

**Pull request merged**